src/context/extensions/steps/getExtensions.js

```diff
@@ -191,6 +192,19 @@ function getCompleteExtensionTree(type, roc, path, initialState) {
     );
 }
 
+function getNearestParents(roc) {
+    return [
+        ...(roc.packages || []),
+        ...(roc.plugins || []),
+    ].map((parent) => {
```

Comment:
I think it's clearer if I say [npm package resolution algorithm](https://nodejs.org/docs/v0.4.8/api/all.html#all_Together...).